Source/Blazorise/Utils/Converters.cs

```diff
@@ -1,156 +1,227 @@
-                    return (float.MinValue, float.MaxValue);
-                case Type doubleType when doubleType == typeof( double ) || doubleType == typeof( double? ):
-                    return (double.MinValue, double.MaxValue);
-                case Type decimalType when decimalType == typeof( decimal ) || decimalType == typeof( decimal? ):
-                    return (decimal.MinValue, decimal.MaxValue);
-                case Type sbyteType when sbyteType == typeof( sbyte ) || sbyteType == typeof( sbyte? ):
-                    return (sbyte.MinValue, sbyte.MaxValue);
-                case Type ushortType when ushortType == typeof( ushort ) || ushortType == typeof( ushort? ):
-                    return (ushort.MinValue, ushort.MaxValue);
-                case Type uintType when uintType == typeof( uint ) || uintType == typeof( uint? ):
-                    return (uint.MinValue, uint.MaxValue);
-                case Type ulongType when ulongType == typeof( ulong ) || ulongType == typeof( ulong? ):
-                    return (ulong.MinValue, ulong.MaxValue);
-                default:
-                    throw new InvalidOperationException( $"Unsupported type {type}" );
-            }
-        }
-    }
-}
+﻿#region Using directives
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Linq;
+#endregion
+
+namespace Blazorise.Utils
+{
+    public static class Converters
+    {
+        #region Constants
+
+        private static readonly Type[] SimpleTypes =
+        {
+            typeof(string),
+            typeof(decimal),
+            typeof(DateTime),
+            typeof(DateTimeOffset),
+            typeof(TimeSpan),
+            typeof(Guid)
+        };
+
+        #endregion
+
+        #region Methods
+
+        /// <summary>
+        /// Converts an object to a dictionary object without the properties which have a null value.
+        /// This can be used as a workaround for System.Text.Json which will always serialize null values which breaks ChartJS functionality.
+        /// </summary>
+        /// <param name="source">The source object, can be null.</param>
+        /// <param name="addEmptyObjects">If this is set to true, objects which do not have any properties are also added to the dictionary. If false, these objects are skipped.</param>
+        /// <returns>Dictionary</returns>
+        public static IDictionary<string, object> ToDictionary( object source, bool addEmptyObjects = true )
+        {
+            if ( source == null )
+            {
+                return null;
+            }
+
+            var dictionary = new Dictionary<string, object>();
+
+            foreach ( PropertyDescriptor property in TypeDescriptor.GetProperties( source ) )
+            {
+                var value = property.GetValue( source );
+
+                if ( value != null )
+                {
+                    var type = value.GetType();

```

Comment:
Try this for Nullable check `var type = Nullable.GetUnderlyingType( value.GetType() ) ?? value.GetType();`